### ISSUE TITLE
Custom Player Skin Confusing

### ISSUE THREAD

**Issue by superblack4** [Bara/TroubleinTerroristTown#314]
So basically I have edit the model.ini document in addon/sourcemod/config/ttt/ and i have enable the model. However, there is nothing happen to player model. How to fix/use this plugin?

**Comment by Bara** [comment]
Can't help you without any informations like the ini file

**Comment by Bara** [comment]
Remove the "//" of every line with your skins.

**Comment by Bara** [comment]
I've updated the default [models.ini](https://github.com/Bara/TroubleinTerroristTown/blob/master/addons/sourcemod/configs/ttt/models.ini) should be more clarify

**Comment by superblack4** [comment]
so the model will appear when player got their role?

**Comment by Bara** [comment]
Exactly, Models will be set with [TTT_OnGetClientRole](https://github.com/Bara/TroubleinTerroristTown/blob/master/addons/sourcemod/scripting/ttt/ttt_models.sp#L271). If you change the role while the round, model should be updated too.

**Comment by superblack4** [comment]
Hi Bara. The model is still not appear even i delete the // and there is no error log even there are player model in my server and FastDL (FastDL is working perfectly)

**Comment by superblack4** [comment]
And can you include arm model in the model too? ✌️

**Comment by Bara** [comment]
Arms support added, not tested but should work

**Comment by Bara** [comment]
Try this version, logs are saved in a own file (addons/sourcemod/logs/ttt_models_<DATE>.log).
[ttt_models.zip](https://github.com/Bara/TroubleinTerroristTown/files/1967664/ttt_models.zip)

**Comment by superblack4** [comment]
L 05/02/2018 - 21:41:17: [ttt_models.smx] DModel1: models/player/custom_player/legacy/security
L 05/02/2018 - 21:41:17: [ttt_models.smx] [ERROR] (OnMapStart) Can't find DModel1 dir: models/player/custom_player/legacy/security
L 05/02/2018 - 21:41:17: [ttt_models.smx] (OnMapStart) DMaterial1: materials/models/player/custom/security
L 05/02/2018 - 21:41:17: [ttt_models.smx] [ERROR] (OnMapStart) Can't find DMaterial1 dir: materials/models/player/custom/security
L 05/02/2018 - 21:41:17: [ttt_models.smx] [ERROR] (OnMapStart) Can't find ITModel1 dir: models/player/custom_player/legacy/prisioner
L 05/02/2018 - 21:41:17: [ttt_models.smx] [ERROR] (OnMapStart) Can't find ITMaterial1 dir: materials/models/player/custom/prisioner
L 05/02/2018 - 21:41:17: [ttt_models.smx] (OnMapStart) AddDownload: models/player/custom_player/legacy/tm_balkan_variant_guerilla.dx90.vtx
L 05/02/2018 - 21:41:17: [ttt_models.smx] (OnMapStart) Precache: models/player/custom_player/legacy/tm_balkan_variant_guerilla.mdl
L 05/02/2018 - 21:41:17: [ttt_models.smx] (OnMapStart) AddDownload: models/player/custom_player/legacy/tm_balkan_variant_guerilla.mdl
L 05/02/2018 - 21:41:17: [ttt_models.smx] (OnMapStart) AddDownload: models/player/custom_player/legacy/tm_balkan_variant_guerilla.phy
L 05/02/2018 - 21:41:17: [ttt_models.smx] (OnMapStart) AddDownload: models/player/custom_player/legacy/tm_balkan_variant_guerilla.vvd
L 05/02/2018 - 21:41:17: [ttt_models.smx] (OnMapStart) AddDownload: materials/models/player/tm_guerilla/tm_guerilla_head.vmt
L 05/02/2018 - 21:41:17: [ttt_models.smx] (OnMapStart) AddDownload: materials/models/player/tm_guerilla/tm_guerilla_head.vtf
L 05/02/2018 - 21:41:17: [ttt_models.smx] (OnMapStart) AddDownload: materials/models/player/tm_guerilla/tm_guerilla_head_normal.vtf
L 05/02/2018 - 21:41:17: [ttt_models.smx] (OnMapStart) AddDownload: materials/models/player/tm_guerilla/tm_guerilla_lowerbody.vmt
L 05/02/2018 - 21:41:17: [ttt_models.smx] (OnMapStart) AddDownload: materials/models/player/tm_guerilla/tm_guerilla_lowerbody.vtf
L 05/02/2018 - 21:41:17: [ttt_models.smx] (OnMapStart) AddDownload: materials/models/player/tm_guerilla/tm_guerilla_upperbody.vmt
L 05/02/2018 - 21:41:17: [ttt_models.smx] (OnMapStart) AddDownload: materials/models/player/tm_guerilla/tm_guerilla_upperbody.vtf
L 05/02/2018 - 21:41:22: [ttt_models.smx] DModel1: models/player/custom_player/legacy/security
L 05/02/2018 - 21:41:22: [ttt_models.smx] [ERROR] (OnMapStart) Can't find DModel1 dir: models/player/custom_player/legacy/security
L 05/02/2018 - 21:41:22: [ttt_models.smx] (OnMapStart) DMaterial1: materials/models/player/custom/security
L 05/02/2018 - 21:41:22: [ttt_models.smx] [ERROR] (OnMapStart) Can't find DMaterial1 dir: materials/models/player/custom/security
L 05/02/2018 - 21:41:22: [ttt_models.smx] [ERROR] (OnMapStart) Can't find ITModel1 dir: models/player/custom_player/legacy/prisioner
L 05/02/2018 - 21:41:22: [ttt_models.smx] [ERROR] (OnMapStart) Can't find ITMaterial1 dir: materials/models/player/custom/prisioner
L 05/02/2018 - 21:41:22: [ttt_models.smx] (OnMapStart) AddDownload: models/player/custom_player/legacy/tm_balkan_variant_guerilla.dx90.vtx
L 05/02/2018 - 21:41:22: [ttt_models.smx] (OnMapStart) Precache: models/player/custom_player/legacy/tm_balkan_variant_guerilla.mdl
L 05/02/2018 - 21:41:22: [ttt_models.smx] (OnMapStart) AddDownload: models/player/custom_player/legacy/tm_balkan_variant_guerilla.mdl
L 05/02/2018 - 21:41:23: [ttt_models.smx] (OnMapStart) AddDownload: models/player/custom_player/legacy/tm_balkan_variant_guerilla.phy
L 05/02/2018 - 21:41:23: [ttt_models.smx] (OnMapStart) AddDownload: models/player/custom_player/legacy/tm_balkan_variant_guerilla.vvd
L 05/02/2018 - 21:41:23: [ttt_models.smx] (OnMapStart) AddDownload: materials/models/player/tm_guerilla/tm_guerilla_head.vmt
L 05/02/2018 - 21:41:23: [ttt_models.smx] (OnMapStart) AddDownload: materials/models/player/tm_guerilla/tm_guerilla_head.vtf
L 05/02/2018 - 21:41:23: [ttt_models.smx] (OnMapStart) AddDownload: materials/models/player/tm_guerilla/tm_guerilla_head_normal.vtf
L 05/02/2018 - 21:41:23: [ttt_models.smx] (OnMapStart) AddDownload: materials/models/player/tm_guerilla/tm_guerilla_lowerbody.vmt
L 05/02/2018 - 21:41:23: [ttt_models.smx] (OnMapStart) AddDownload: materials/models/player/tm_guerilla/tm_guerilla_lowerbody.vtf
L 05/02/2018 - 21:41:23: [ttt_models.smx] (OnMapStart) AddDownload: materials/models/player/tm_guerilla/tm_guerilla_upperbody.vmt
L 05/02/2018 - 21:41:23: [ttt_models.smx] (OnMapStart) AddDownload: materials/models/player/tm_guerilla/tm_guerilla_upperbody.vtf
L 05/02/2018 - 21:42:18: [ttt_models.smx] [ERROR] (SetModel) Can't find ITModel1 dir: models/player/custom_player/legacy/prisioner
L 05/02/2018 - 21:42:18: [ttt_models.smx] Player: Kun|csgofast.com|farmskins.com, Model: models/player/custom_player/legacy/tm_balkan_variant_guerilla.mdl
L 05/02/2018 - 21:42:18: [ttt_models.smx] [ERROR] (SetModel) Can't find ITModel1 dir: models/player/custom_player/legacy/prisioner
L 05/02/2018 - 21:42:18: [ttt_models.smx] Player: ★LordCat★, Model: models/player/custom_player/legacy/tm_balkan_variant_guerilla.mdl
L 05/02/2018 - 21:42:18: [ttt_models.smx] Player: ｔｒｅｅ, Model: models/player/custom_player/legacy/tm_balkan_variant_guerilla.mdl
L 05/02/2018 - 21:44:00: [ttt_models.smx] Player: Gin, Model: models/player/custom_player/legacy/tm_balkan_variant_guerilla.mdl
L 05/02/2018 - 21:44:01: [ttt_models.smx] [ERROR] (SetModel) Can't find ITModel1 dir: models/player/custom_player/legacy/prisioner
L 05/02/2018 - 21:44:01: [ttt_models.smx] [ERROR] (SetModel) Can't find DModel1 dir: models/player/custom_player/legacy/security
L 05/02/2018 - 21:44:01: [ttt_models.smx] Player: ★LordCat★, Model: models/player/custom_player/legacy/tm_balkan_variant_guerilla.mdl
L 05/02/2018 - 21:44:01: [ttt_models.smx] Player: ｔｒｅｅ, Model: models/player/custom_player/legacy/tm_balkan_variant_guerilla.mdl
L 05/02/2018 - 21:44:01: [ttt_models.smx] [ERROR] (SetModel) Can't find DModel1 dir: models/player/custom_player/legacy/security
L 05/02/2018 - 21:44:01: [ttt_models.smx] [ERROR] (SetModel) Can't find ITModel1 dir: models/player/custom_player/legacy/prisioner
L 05/02/2018 - 21:44:01: [ttt_models.smx] [ERROR] (SetModel) Can't find ITModel1 dir: models/player/custom_player/legacy/prisioner
L 05/02/2018 - 21:45:28: [ttt_models.smx] [ERROR] (SetModel) Can't find ITModel1 dir: models/player/custom_player/legacy/prisioner
L 05/02/2018 - 21:45:28: [ttt_models.smx] [ERROR] (SetModel) Can't find ITModel1 dir: models/player/custom_player/legacy/prisioner
L 05/02/2018 - 21:45:28: [ttt_models.smx] [ERROR] (SetModel) Can't find ITModel1 dir: models/player/custom_player/legacy/prisioner
L 05/02/2018 - 21:45:28: [ttt_models.smx] [ERROR] (SetModel) Can't find ITModel1 dir: models/player/custom_player/legacy/prisioner
L 05/02/2018 - 21:45:28: [ttt_models.smx] [ERROR] (SetModel) Can't find DModel1 dir: models/player/custom_player/legacy/security
L 05/02/2018 - 21:45:28: [ttt_models.smx] [ERROR] (SetModel) Can't find DModel1 dir: models/player/custom_player/legacy/security
L 05/02/2018 - 21:45:28: [ttt_models.smx] [ERROR] (SetModel) Can't find ITModel1 dir: models/player/custom_player/legacy/prisioner
L 05/02/2018 - 21:45:28: [ttt_models.smx] [ERROR] (SetModel) Can't find ITModel1 dir: models/player/custom_player/legacy/prisioner
L 05/02/2018 - 21:46:52: [ttt_models.smx] Player: JOOOHHHNNN CEEENAAA, Model: models/player/custom_player/legacy/tm_balkan_variant_guerilla.mdl
L 05/02/2018 - 21:46:52: [ttt_models.smx] Player: ★LordCat★, Model: models/player/custom_player/legacy/tm_balkan_variant_guerilla.mdl
L 05/02/2018 - 21:46:52: [ttt_models.smx] Player: ｔｒｅｅ, Model: models/player/custom_player/legacy/tm_balkan_variant_guerilla.mdl
L 05/02/2018 - 21:46:52: [ttt_models.smx] [ERROR] (SetModel) Can't find ITModel1 dir: models/player/custom_player/legacy/prisioner
L 05/02/2018 - 21:46:52: [ttt_models.smx] Player: Wolf, Model: models/player/custom_player/legacy/tm_balkan_variant_guerilla.mdl
L 05/02/2018 - 21:46:52: [ttt_models.smx] [ERROR] (SetModel) Can't find DModel1 dir: models/player/custom_player/legacy/security
L 05/02/2018 - 21:46:52: [ttt_models.smx] Player: Hongtuky, Model: models/player/custom_player/legacy/tm_balkan_variant_guerilla.mdl
L 05/02/2018 - 21:48:37: [ttt_models.smx] [ERROR] (SetModel) Can't find ITModel1 dir: models/player/custom_player/legacy/prisioner
L 05/02/2018 - 21:48:37: [ttt_models.smx] [ERROR] (SetModel) Can't find DModel1 dir: models/player/custom_player/legacy/security
L 05/02/2018 - 21:48:37: [ttt_models.smx] [ERROR] (SetModel) Can't find ITModel1 dir: models/player/custom_player/legacy/prisioner
L 05/02/2018 - 21:48:37: [ttt_models.smx] Player: Mr.Warriors, Model: models/player/custom_player/legacy/tm_balkan_variant_guerilla.mdl
L 05/02/2018 - 21:48:37: [ttt_models.smx] [ERROR] (SetModel) Can't find ITModel1 dir: models/player/custom_player/legacy/prisioner
L 05/02/2018 - 21:48:37: [ttt_models.smx] Player: R- be back on July [probably], Model: models/player/custom_player/legacy/tm_balkan_variant_guerilla.mdl
L 05/02/2018 - 21:48:37: [ttt_models.smx] [ERROR] (SetModel) Can't find ITModel1 dir: models/player/custom_player/legacy/prisioner
L 05/02/2018 - 21:50:43: [ttt_models.smx] [ERROR] (SetModel) Can't find ITModel1 dir: models/player/custom_player/legacy/prisioner
L 05/02/2018 - 21:50:43: [ttt_models.smx] [ERROR] (SetModel) Can't find ITModel1 dir: models/player/custom_player/legacy/prisioner
L 05/02/2018 - 21:50:43: [ttt_models.smx] Player: ｔｒｅｅ, Model: models/player/custom_player/legacy/tm_balkan_variant_guerilla.mdl
L 05/02/2018 - 21:50:43: [ttt_models.smx] Player: Mr.Warriors, Model: models/player/custom_player/legacy/tm_balkan_variant_guerilla.mdl
L 05/02/2018 - 21:50:43: [ttt_models.smx] Player: Wolf, Model: models/player/custom_player/legacy/tm_balkan_variant_guerilla.mdl
L 05/02/2018 - 21:50:43: [ttt_models.smx] Player: R- be back on July [probably], Model: models/player/custom_player/legacy/tm_balkan_variant_guerilla.mdl
L 05/02/2018 - 21:50:43: [ttt_models.smx] [ERROR] (SetModel) Can't find DModel1 dir: models/player/custom_player/legacy/security
L 05/02/2018 - 21:52:05: [ttt_models.smx] [ERROR] (SetModel) Can't find ITModel1 dir: models/player/custom_player/legacy/prisioner
L 05/02/2018 - 21:52:05: [ttt_models.smx] Player: ★LordCat★, Model: models/player/custom_player/legacy/tm_balkan_variant_guerilla.mdl
L 05/02/2018 - 21:52:05: [ttt_models.smx] [ERROR] (SetModel) Can't find ITModel1 dir: models/player/custom_player/legacy/prisioner
L 05/02/2018 - 21:52:05: [ttt_models.smx] [ERROR] (SetModel) Can't find DModel1 dir: models/player/custom_player/legacy/security
L 05/02/2018 - 21:52:05: [ttt_models.smx] Player: Wolf, Model: models/player/custom_player/legacy/tm_balkan_variant_guerilla.mdl
L 05/02/2018 - 21:52:05: [ttt_models.smx] Player: R- be back on July [probably], Model: models/player/custom_player/legacy/tm_balkan_variant_guerilla.mdl
L 05/02/2018 - 21:52:05: [ttt_models.smx] [ERROR] (SetModel) Can't find ITModel1 dir: models/player/custom_player/legacy/prisioner
L 05/02/2018 - 21:54:43: [ttt_models.smx] [ERROR] (SetModel) Can't find ITModel1 dir: models/player/custom_player/legacy/prisioner
L 05/02/2018 - 21:54:43: [ttt_models.smx] [ERROR] (SetModel) Can't find DModel1 dir: models/player/custom_player/legacy/security
L 05/02/2018 - 21:54:43: [ttt_models.smx] [ERROR] (SetModel) Can't find ITModel1 dir: models/player/custom_player/legacy/prisioner
L 05/02/2018 - 21:54:43: [ttt_models.smx] [ERROR] (SetModel) Can't find ITModel1 dir: models/player/custom_player/legacy/prisioner
L 05/02/2018 - 21:54:43: [ttt_models.smx] [ERROR] (SetModel) Can't find ITModel1 dir: models/player/custom_player/legacy/prisioner
L 05/02/2018 - 21:54:43: [ttt_models.smx] Player: R- be back on July [probably], Model: models/player/custom_player/legacy/tm_balkan_variant_guerilla.mdl
L 05/02/2018 - 21:54:43: [ttt_models.smx] [ERROR] (SetModel) Can't find ITModel1 dir: models/player/custom_player/legacy/prisioner
L 05/02/2018 - 21:57:04: [ttt_models.smx] [ERROR] (SetModel) Can't find ITModel1 dir: models/player/custom_player/legacy/prisioner
L 05/02/2018 - 21:57:04: [ttt_models.smx] [ERROR] (SetModel) Can't find ITModel1 dir: models/player/custom_player/legacy/prisioner
L 05/02/2018 - 21:57:04: [ttt_models.smx] [ERROR] (SetModel) Can't find ITModel1 dir: models/player/custom_player/legacy/prisioner
L 05/02/2018 - 21:57:04: [ttt_models.smx] [ERROR] (SetModel) Can't find ITModel1 dir: models/player/custom_player/legacy/prisioner
L 05/02/2018 - 21:57:04: [ttt_models.smx] Player: Mr.Warriors, Model: models/player/custom_player/legacy/tm_balkan_variant_guerilla.mdl
L 05/02/2018 - 21:57:04: [ttt_models.smx] [ERROR] (SetModel) Can't find ITModel1 dir: models/player/custom_player/legacy/prisioner
L 05/02/2018 - 21:57:04: [ttt_models.smx] [ERROR] (SetModel) Can't find DModel1 dir: models/player/custom_player/legacy/security
L 05/02/2018 - 21:59:00: [ttt_models.smx] [ERROR] (SetModel) Can't find ITModel1 dir: models/player/custom_player/legacy/prisioner
L 05/02/2018 - 21:59:00: [ttt_models.smx] Player: ✪︵pok3r farmskins.com, Model: models/player/custom_player/legacy/tm_balkan_variant_guerilla.mdl
L 05/02/2018 - 21:59:00: [ttt_models.smx] [ERROR] (SetModel) Can't find ITModel1 dir: models/player/custom_player/legacy/prisioner
L 05/02/2018 - 21:59:00: [ttt_models.smx] [ERROR] (SetModel) Can't find DModel1 dir: models/player/custom_player/legacy/security
L 05/02/2018 - 21:59:00: [ttt_models.smx] Player: Mr.Warriors, Model: models/player/custom_player/legacy/tm_balkan_variant_guerilla.mdl
L 05/02/2018 - 21:59:00: [ttt_models.smx] Player: Wolf, Model: models/player/custom_player/legacy/tm_balkan_variant_guerilla.mdl
L 05/02/2018 - 21:59:00: [ttt_models.smx] Player: R- be back on July [probably], Model: models/player/custom_player/legacy/tm_balkan_variant_guerilla.mdl
L 05/02/2018 - 22:01:32: [ttt_models.smx] [ERROR] (SetModel) Can't find DModel1 dir: models/player/custom_player/legacy/security
L 05/02/2018 - 22:01:32: [ttt_models.smx] [ERROR] (SetModel) Can't find ITModel1 dir: models/player/custom_player/legacy/prisioner
L 05/02/2018 - 22:01:32: [ttt_models.smx] [ERROR] (SetModel) Can't find ITModel1 dir: models/player/custom_player/legacy/prisioner
L 05/02/2018 - 22:01:32: [ttt_models.smx] [ERROR] (SetModel) Can't find ITModel1 dir: models/player/custom_player/legacy/prisioner
L 05/02/2018 - 22:01:32: [ttt_models.smx] Player: Mr.Warriors, Model: models/player/custom_player/legacy/tm_balkan_variant_guerilla.mdl
L 05/02/2018 - 22:01:32: [ttt_models.smx] [ERROR] (SetModel) Can't find ITModel1 dir: models/player/custom_player/legacy/prisioner
L 05/02/2018 - 22:01:32: [ttt_models.smx] Player: R- be back on July [probably], Model: models/player/custom_player/legacy/tm_balkan_variant_guerilla.mdl
L 05/02/2018 - 22:01:32: [ttt_models.smx] [ERROR] (SetModel) Can't find DModel1 dir: models/player/custom_player/legacy/security
L 05/02/2018 - 22:01:32: [ttt_models.smx] Player: JOOOHHHNNN CEEENAAA, Model: models/player/custom_player/legacy/tm_balkan_variant_guerilla.mdl
L 05/02/2018 - 22:03:24: [ttt_models.smx] Player: Gin, Model: models/player/custom_player/legacy/tm_balkan_variant_guerilla.mdl
L 05/02/2018 - 22:03:24: [ttt_models.smx] [ERROR] (SetModel) Can't find ITModel1 dir: models/player/custom_player/legacy/prisioner
L 05/02/2018 - 22:03:24: [ttt_models.smx] Player: ♥ Dio Brando ♥, Model: models/player/custom_player/legacy/tm_balkan_variant_guerilla.mdl
L 05/02/2018 - 22:03:24: [ttt_models.smx] [ERROR] (SetModel) Can't find DModel1 dir: models/player/custom_player/legacy/security
L 05/02/2018 - 22:03:24: [ttt_models.smx] Player: Mr.Warriors, Model: models/player/custom_player/legacy/tm_balkan_variant_guerilla.mdl
L 05/02/2018 - 22:03:24: [ttt_models.smx] Player: Wolf, Model: models/player/custom_player/legacy/tm_balkan_variant_guerilla.mdl
L 05/02/2018 - 22:03:24: [ttt_models.smx] Player: R- be back on July [probably], Model: models/player/custom_player/legacy/tm_balkan_variant_guerilla.mdl
L 05/02/2018 - 22:03:24: [ttt_models.smx] [ERROR] (SetModel) Can't find DModel1 dir: models/player/custom_player/legacy/security
L 05/02/2018 - 22:03:24: [ttt_models.smx] [ERROR] (SetModel) Can't find ITModel1 dir: models/player/custom_player/legacy/prisioner
L 05/02/2018 - 22:03:24: [ttt_models.smx] [ERROR] (SetModel) Can't find ITModel1 dir: models/player/custom_player/legacy/prisioner
L 05/02/2018 - 22:03:24: [ttt_models.smx] [ERROR] (SetModel) Can't find ITModel1 dir: models/player/custom_player/legacy/prisioner
L 05/02/2018 - 22:07:00: [ttt_models.smx] Player: Gin, Model: models/player/custom_player/legacy/tm_balkan_variant_guerilla.mdl
L 05/02/2018 - 22:07:00: [ttt_models.smx] [ERROR] (SetModel) Can't find ITModel1 dir: models/player/custom_player/legacy/prisioner
L 05/02/2018 - 22:07:00: [ttt_models.smx] Player: ♥ Dio Brando ♥, Model: models/player/custom_player/legacy/tm_balkan_variant_guerilla.mdl
L 05/02/2018 - 22:07:00: [ttt_models.smx] [ERROR] (SetModel) Can't find ITModel1 dir: models/player/custom_player/legacy/prisioner
L 05/02/2018 - 22:07:01: [ttt_models.smx] [ERROR] (SetModel) Can't find DModel1 dir: models/player/custom_player/legacy/security
L 05/02/2018 - 22:07:01: [ttt_models.smx] Player: Wolf, Model: models/player/custom_player/legacy/tm_balkan_variant_guerilla.mdl
L 05/02/2018 - 22:07:01: [ttt_models.smx] [ERROR] (SetModel) Can't find ITModel1 dir: models/player/custom_player/legacy/prisioner
L 05/02/2018 - 22:07:01: [ttt_models.smx] Player: ۩ Ｍ❂ñ ♬, Model: models/player/custom_player/legacy/tm_balkan_variant_guerilla.mdl
L 05/02/2018 - 22:07:01: [ttt_models.smx] [ERROR] (SetModel) Can't find DModel1 dir: models/player/custom_player/legacy/security
L 05/02/2018 - 22:07:01: [ttt_models.smx] [ERROR] (SetModel) Can't find DModel1 dir: models/player/custom_player/legacy/security
L 05/02/2018 - 22:07:01: [ttt_models.smx] [ERROR] (SetModel) Can't find ITModel1 dir: models/player/custom_player/legacy/prisioner
L 05/02/2018 - 22:07:01: [ttt_models.smx] Player: Dory |><(((º> | 1 deag, Model: models/player/custom_player/legacy/tm_balkan_variant_guerilla.mdl

Ok here is the log. Can u check if i make any mistake?

**Comment by superblack4** [comment]
My Model.ini

"TTT-Models"
{
	"Models"
	{
		// Do you want this model feature?
		"Enable"				"1"

		// How many detective models are there?
		"DModelCount"			"1"

		// How many innocent/traitor models are there?
		"ITModelCount"			"2"

		// Detective models (THIS IS JUST A MODEL AS EXAMPLE!)
		"DModel1"				"models/player/custom_player/legacy/security"
		"DMaterial1"			"materials/models/player/custom/security"

		// Innocent/Traitor models (THIS IS JUST A MODEL AS EXAMPLE!)
		"ITModel1"				"models/player/custom_player/legacy/prisioner"
		"ITMaterial1"			"materials/models/player/custom/prisioner"

		// Example(s)
		// If you want more models increase the number at the end and update DModelCount and/or ITModelCount like this:
		// ALL MODELS ARE JUST EXAMPLES
		// "DModelCount"			"1"
		// "ITModelCount"			"2"

		 "DModel1"				"models/player/custom_player/msgs4/frog"
		 "DMaterial1"			"materials/models/player/msgs4/frog"
		
		 "ITModel1"				"models/player/custom_player/kirby/umbrella_leet"
		 "ITMaterial1"			"materials/models/player/kirby/umbrella_leet"
		 "ITModel2"				"models/player/custom_player/legacy"
		 "ITMaterial2"			"materials/models/player/tm_guerilla"
	}
}

**Comment by superblack4** [comment]
the guerilla player model are somehow working but not Leet and Frog model. This is weird ✌️

**Comment by Bara** [comment]
Upload (Drag&Drop) the files instead of Copy&Paste (**better to read**)

**Comment by superblack4** [comment]
Ok let me try again Bara

**Comment by Bara** [comment]
(I mean the log file + models.ini) ;)

**Comment by superblack4** [comment]
[Uploading models_18-05-02.log…]()



[models.txt](https://github.com/Bara/TroubleinTerroristTown/files/1968044/models.txt)

**Comment by superblack4** [comment]
I have delete the leet model since it has bug, however the detective did not get the skin (Frog model) but Innocent got it (Leet model)

**Comment by Bara** [comment]
I think you was too fast for your upload (log file wasn't uploaded). 😄

**Comment by superblack4** [comment]
Damm i am sorry. Let me reupload it again :(

**Comment by superblack4** [comment]
[models_18-05-02.log](https://github.com/Bara/TroubleinTerroristTown/files/1968074/models_18-05-02.log)

**Comment by Bara** [comment]
Are this the default models, if not any download links? First look i see no problems... could you try this one (better logging): [ttt_models.zip](https://github.com/Bara/TroubleinTerroristTown/files/1968170/ttt_models.zip)


I've updated your models.ini (bit cleaner): [models.txt](https://github.com/Bara/TroubleinTerroristTown/files/1968145/models.txt)

**Comment by superblack4** [comment]
https://gamebanana.com/skins/156759 (Leet Model which is bug)
https://gamebanana.com/skins/153338 (Detective model, not working for detective)
https://gamebanana.com/skins/145754 (Guerilla Model which is working)

**Comment by superblack4** [comment]
I will give it a try tomorrow and tell u the result. Thank you so much for your effort @Bara :)

**Comment by superblack4** [comment]
[models_18-05-03.log](https://github.com/Bara/TroubleinTerroristTown/files/1970963/models_18-05-03.log)
Hi Bara. I have the test and the result is weird. All the model all now disappear :+1:

**Comment by superblack4** [comment]
[models.txt](https://github.com/Bara/TroubleinTerroristTown/files/1970964/models.txt)
I have 2 new skin and they all disappear. :)

**Comment by superblack4** [comment]
@Bara Good luck with that my friend :)

**Comment by superblack4** [comment]
@Bara Did you find the solution to fix the skin that is disappear?

**Comment by Bara** [comment]
So, I'm nearly done and it looks "better" now. Debug Logs should be more helpful than before.

Screen 1: https://steamcommunity.com/sharedfiles/filedetails/?id=1379550254
Screen 2: https://steamcommunity.com/sharedfiles/filedetails/?id=1379550318

Model pack: https://gamebanana.com/skins/154984
Debug log: [ttt_models_1525617005.log](https://github.com/Bara/TroubleinTerroristTown/files/1977702/ttt_models_1525617005.log)

**Comment by superblack4** [comment]
@bara 
1. Did you finish the plugin?
2. Can i see your own model.ini please?

Thx you so much for your hard work

**Comment by Bara** [comment]
@superblack4 1. Yes, should be done 2. You can find [all files here](https://github.com/Bara/TroubleinTerroristTown/issues/314#issuecomment-386884084).

**Comment by superblack4** [comment]
OK. So i have test your new one and it seem to work like a charm. However, only the first skin of Detective team and Inocent are appear. Not the second or third one. (I have install new skin player for my server)
Model pack: https://gamebanana.com/skins/150025

Screenshot: http://steamcommunity.com/profiles/76561198097534771/screenshot/922553336798255688

Models that are appear:
Detective: Gign
Innocent: Carrie 

Models that not appear: 
Detectives: SAS, 707
Innocents: Krachenko, Lincoln
 
[model.txt](https://github.com/Bara/TroubleinTerroristTown/files/1980035/model.txt)

![cso2pack2](https://user-images.githubusercontent.com/38873974/39704770-9af22b34-5236-11e8-976b-72cb0a179cf1.jpg)

**Comment by Bara** [comment]
You should uncomment ( remove // )this lines:
>		// "DModelCount"			"3"
>		// "ITModelCount"			"3"

**Comment by superblack4** [comment]
OMG @Bara  U are my god. Thx you so much, The models are now working like a charm. Any won't you mind if I ask you about how to add gun on NON TTT map?

**Comment by Bara** [comment]
https://forums.alliedmods.net/showthread.php?p=2487944
but i prefer to create a new issue for this.

**Comment by superblack4** [comment]
@Bara when i spawn the weapon for the first time, I can see the weapon. But when I restart the server, the weapon are gone

Database: 
"hl_weaponspawner"
	{
		"driver"			"sqlite"
		"host"				"localhost"
		"database"			"sourcemod"
		"user"				""
		"pass"				""	
		//"timeout"			"0"
		//"port"			"0"
	}
